### ISSUE TITLE
Pressable documentation missing accessibility props

### ISSUE THREAD

**Issue by unitario** [facebook/react-native-website#4773]
## Description

The [`PressableProps`](https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Components/Pressable/Pressable.d.ts) type definition for the `Pressable` component extends [`AccessibilityProps`](https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Components/View/AccessibilityProps.d.ts).

However, this relationship is not documented in the [official `Pressable` documentation](https://reactnative.dev/docs/pressable#props). As a result, the accessibility-related props that are available through `AccessibilityProps` are missing from the docs.

For consistency, I would expect these props to be listed explicitly, similar to how they are documented for the [`View`](https://reactnative.dev/docs/view#props) component.

## Documentation version

`=> 0.70`
